app.js

```diff
@@ -1,16 +1,21 @@
-//dependencies
+//imports
 import createError from 'http-errors';
 import express from 'express';
 import path from 'path';
 import cookieParser from 'cookie-parser';
 import logger from 'morgan';
-import indexRouter from './routes/index.js';
 import { dirname } from 'path';
 import { fileURLToPath } from 'url';
+
+//custom imports
+import indexRouter from './routes/index.js';
+import updateRouter from './routes/update.js';
 import db from'./db/query.js';
 
 const __dirname = dirname(fileURLToPath(import.meta.url));
 const app = express();
+//creates fresh table every time--for development only
+db.dropTable();
```

Comment:
Should this be committed if this is for development only? If so, this should live under a flag.